### ISSUE TITLE
Grafana port incorrect?

### ISSUE THREAD

**Issue by theweekendgeek** [stefanprodan/dockprom#92]
Edit: So upon further inspection I saw that the port is correct, bc. of the way caddy works. So that is not the issue, I still can not figure out how to get rid of the 502 error, or that causes it.

I am using this with the docker toolbox on windows 7.

In the readme it says to "Navigate to http://<host-ip>:3000" and login into grafana. 

I`'ve cloned the repo and as it stands the container caddy is mapped to the host port 3000. Accessing that gives me a 502 error.

(Additionally, I've managed to set grafana another port via Kitematic and successfully log into grafana that way but that seems to also conflict with caddy and I can't find any prometheus datasource preconfigured. 
I'm not sure however this might be somehow related to this issue in some way. I can, however, access prometheus and access stats and metrics for my containers)

**Comment by pazpi** [comment]
I have your same problem, when connecting to http://localhost:3000 I receive a 502 Bad Gateway error.
Inspecting the log I found that Grafana hasn't the right permission to write to the database, adding
`user: "104"` in `docker-compose.yml` after the port section solved this issue.

I found the solution here, but I don't know if it is a temporal fix.
[http://docs.grafana.org/installation/docker/#migration-from-a-previous-version-of-the-docker-container-to-5-1-or-later](url)
